control/planktoscopehat/planktoscope/camera/mqtt.py

```diff
@@ -107,8 +104,19 @@ def run(self) -> None:
                 if (message := mqtt.msg) is None:
                     continue
                 self._receive_message(message)
-                if (status_update := mqtt.read_message()) is not None:
-                    mqtt.client.publish("status/imager", status_update)
+                if message["topic"] == "camera/info" and message["payload"].get("action") == "get":
+                    camera_name = (
+                        self._camera.camera_name
+                        if self._camera is not None
+                        else "Not recognized"
+                    )
+                    response_payload = json.dumps(
+                        {"status": "success", "camera_name": camera_name}
+                    )
+                    mqtt.client.publish("status/camera/info", response_payload)
+                    loguru.logger.info(
+                        f"Published camera name '{camera_name}' to topic 'status/camera/info'"
+                    )
```

Comment:
This log message is not very useful. Let's change it from the `info` log level to the `debug` log level.

Reply:
I changed this comment back to "unresolved" because it's still at the `info` log level, and I still think it should be at the `debug` log level. If think think there's a good reason for it to remain at the `info` log level, please describe your reasoning for it - and then I will mark this comment as "resolved".